harvester.go

```diff
@@ -103,15 +103,15 @@ func (b *Builder) Create() (Harvester, error) {
 	if b.err != nil {
 		return nil, b.err
 	}
-	seed := seed.New(b.seedParams...)
+	sd := seed.New(b.seedParams...)
```

Comment:
By reading this, I think that a new `seed` value is created, but it actually creates a `seeder`. Maybe we could change constructor name to `NewSeeder` instead of just `New`.
Another approach could be to change the whole package name from `seed` to `seeder`.

Reply:
You are correct, but:
- Package names should describe what is in it. Seeder is too specific.
- ```seed.NewSeeder()``` introduces stuttering 'seed' which is also not ideal.
I am open to other suggestions.

Reply:
I agree on both comments but, the first doesn't apply here.
`seed.go` has a `Seeder` that `Seed()` something, the same way `watcher.go` has a `Watcher` that `Watch()` something.

Reply:
I don't follow. watcher lives in the monitor package, not in the `watch` package to be comparable.

Reply:
```
Package names should describe what is in it. Seeder is too specific.
```
You are right. I wasn't clear enough.
My point was that since package contains a `seeder` and not a `seed`, I don't think naming the package `seeder` is too specific.

Reply:
Naming things is hard. Yes, I get your point.

Reply:
We will keep current structure as it is, until a better and commonly approved solution is proposed.